recipes-bsp/u-boot/files/0025-gpio-da8xx_gpio-Fix-gpio-name-with-address.patch

```diff
@@ -0,0 +1,42 @@
+From 709cd7d1e433326654e048096e010ff72a1d90e0 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Fri, 7 Jan 2022 10:03:56 +0800
+Subject: [PATCH 25/28] gpio: da8xx_gpio: Fix gpio name with address
+
+THe GPIO bank numbers do not appear in the device tree,
```

Comment:
Typo: "THe"

It's better to cherry-pick this from upstream where it was already applied.

Reply:
has picked it from upstream.fix the issue.close the conversation